### PR TITLE
[CPT-2019] Adding program, Karl. Cleaning up images.

### PR DESCRIPTION
**Pull request by cobusbernard**
Added Karl as speaker.
Added a program.
Cleanup speaker images, made them square.

### CODE REVIEW

**Comment by cobusbernard** [comment]
DO NOT MERGE YET - Had issues with the generated program, want to check on the CI branch.

**Comment by MikeRosTX** [comment]
Marked **do-not-merge** until you give use the 👍

File: content/events/2019-cape-town/program.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
++++
+title = "program"
+type = "program"
+Description = "Program for DevOpsDays Cape Town 2018"
```
Comment (adrianmoisey) [inline content/events/2019-cape-town/program.md]: 2019

File: content/events/2019-cape-town/speakers/gareth-mccumskey.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 +++
-Title = "Gareth Mc Cumskey"
+Title = "Gareth McCumskey"
```
Comment (adrianmoisey) [inline content/events/2019-cape-town/speakers/gareth-mccumskey.md]: It’s Mc Cumskey. I double checked this with Gareth

Comment (cobusbernard) [reply content/events/2019-cape-town/speakers/gareth-mccumskey.md]: Thanks, I had a look at his online profiles and they don't have spaces in.

**Comment by cobusbernard** [comment]
Thanks for fixing @MikeRosTX, we are ready to merge after this last change.

**Pull request merged**